examples/hotel_ads/add_hotel_listing_group_tree.py

```diff
@@ -245,6 +245,12 @@ def _add_level1_nodes(
     other_hotels_listing_dimension_info = client.get_type(
         "ListingDimensionInfo", version="v6"
     )
+    # This is necessary to set "hotel_class" as the oneof field on the
```

Comment:
I don't think that "This is necessary" is necessary :)

Reply:
Done.